css/style.css

```diff
@@ -0,0 +1,17 @@
+.container{
+    width:500px;
```

Comment:
Is there a way of doing this in a more responsive manner? Using percentages perhaps or viewport width / height (vh, vw)...